marimo/_sql/get_engines.py

```diff
@@ -47,10 +48,16 @@ def engine_to_data_source_connection(
 ) -> DataSourceConnection:
     databases: list[Database] = []
     if isinstance(engine, SQLAlchemyEngine):
+        config = (
```

Comment:
you might be able to get this from the `get_context `. you can fallback to `get_default_config_manager`